COLLABORATOR_GUIDE.md

```diff
@@ -167,32 +213,37 @@ export default MyComponent;
 
 - Only spread props `{ ... }` on the definition of the Component (Avoid having a variable named `props`)
 - Avoid importing `React`, only import the modules from React that you need
-- When importing types use `import type { NameOfImport } from 'module'`
-- When defining a Component use the `FC` type from React to define the type of the Component
+- When importing types, use `import type { NameOfImport } from 'module'`
+- When defining a Component, use the `FC` type from React to define the type of the Component
   - When using `children` as a prop, use the `FC<PropsWithChildren<MyComponentProps>>` type instead
   - Alterenatively you can define your type as `type MyComponentProps = PropsWithChildren<{ my other props}>`
 - Each Props type should be prefixed by the name of the Component
 - Components should always be the `default` export of a React Component file
-- Avoid using DOM/Web APIs/`document`/`window` API access within a React Component. Use utilities or Hooks when you need a Reactive state
+- Avoid using DOM/Web APIs/`document`/`window` API access within a React Component.
+  Use utilities or Hooks when you need a Reactive state
 - Avoid making your Component too big. Deconstruct it into smaller Components/Hooks whenever possible
 
 ## Unit Tests and Storybooks
 
-Each new feature or bug fix should be accompanied by a unit test (when deemed valuable). We use [Jest][] as our test runner and [React Testing Library][] for our React unit tests.
+Each new feature or bug fix should accompany a unit test (when deemed valuable).
```

Comment:
This change should be reverted. The rephrasing makes it sound like unit tests come first, and the features or bugs come along as a consequence of the test.

Reply:
I am not sure what you're talking about. I've applied the changes.